tests/TestModel.php

```diff
@@ -14,5 +14,10 @@ class TestModel extends Model
     protected $guarded = [];
     public $timestamps = false;
 
-    public $translatable = ['name', 'other_field'];
+    public $translatable = ['name', 'other_field', 'field_with_mutator'];
+
+    public function setMutableFieldAttribute($value, $lang = null)
```

Comment:
Shouldn't this be updated?

Reply:
Sorry, missed that :disappointed_relieved: